onnxruntime/core/framework/config_options.h

```diff
@@ -32,7 +31,7 @@ struct ConfigOptions {
   std::string GetConfigOrDefault(const std::string& config_key, const std::string& default_value) const noexcept;
 
   // Add a config pair (config_key, config_value) to this instance of ConfigOptions
-  Status AddConfigEntry(const char* config_key, const char* config_value) noexcept;
+  common::Status AddConfigEntry(const char* config_key, const char* config_value) noexcept;
```

Comment:
Is this change necessary?

Reply:
Unless there's a specific reason to make the code more verbose or fix a compile issue I would suggest leaving it as is. We actually do the below so that we don't need to qualify the usage of Status in the onnxruntime domain:

https://github.com/microsoft/onnxruntime/blob/565da712750ea7fd2c56ee71bae84fa557c13aab/include/onnxruntime/core/common/status.h#L192-L193

Reply:
I didn't notice this declaration in the below of status.h.
Removed.